### ISSUE TITLE
brew-file consumes the output of `brew upgrade`

### ISSUE THREAD

**Issue by zcutlip** [rcmdnk/homebrew-file#151]
when I run `brew-file update`, the output of the `brew upgrade` commands seems to be consumed so it doesn't show on the console. This makes it where you don't know what `brew` is doing.

In particular this is a problem when a cask requires the sudo password to upgrade. All you get is the password prompt, but no context for why:

```console
❱ brew file update
$ brew update
Already up-to-date.

$ brew upgrade --formula

$ brew upgrade --cask
Password:
```

Here's the output when I run `\brew upgrade --cask` (bypassing the shell wrapper):

```console
❱ \brew upgrade --cask                                                                                                            
==> Casks with 'auto_updates true' or 'version :latest' will not be upgraded; pass `--greedy` to upgrade them.
==> Upgrading 1 outdated package:
elgato-stream-deck 6.0.4.18165 -> 6.0.5.18168
==> Upgrading elgato-stream-deck
==> Downloading https://edge.elgato.com/egc/macos/sd/Stream_Deck_6.0.5.18168.pkg
Already downloaded: /Users/zach/Library/Caches/Homebrew/downloads/4fed8aa3d15f18bd50b639d04829da01b6fd74f940f38172704e6e2215ff8f14--Stream_Deck_6.0.5.18168.pkg
==> Removing launchctl service com.elgato.StreamDeck
Password:
==> Uninstalling packages; your password may be necessary:
==> Removing files:
/Applications/Elgato Stream Deck.app
==> Running installer for elgato-stream-deck; your password may be necessary.
Package installers may write to any location; options such as `--appdir` are ignored.
installer: Package name is Stream Deck
installer: Installing at base path /
installer: The install was successful.
==> Purging files for version 6.0.4.18165 of Cask elgato-stream-deck
🍺  elgato-stream-deck was successfully upgraded!
```

Here's my brew-file and home-brew version details:

```console
❱ brew-file --version
Homebrew 4.0.4-113-gf3cf599
Homebrew/homebrew-core (git revision 9ec688970ba; last commit 2023-02-20)
Homebrew/homebrew-cask (git revision 0b321ca83f; last commit 2023-02-21)

brew-file 9.0.0 01/Mar/2023
```

**Comment by rcmdnk** [comment]
Thanks for the report!
It should be fixed at v9.0.1. could you please try it?

**Comment by zcutlip** [comment]
Yes it seems to be working now. thank you.
